source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst

```diff
@@ -7,88 +7,82 @@ Robot Preemptive Troubleshooting
 
 Spade connectors can also fail due to improper crimps, so tug-test those as well.
 
-Blade fuses
-------------------------------------
+Blade Fuses
+-----------
 
 .. image:: images/preemptive-troubleshooting/preCheckFuse.png
-   :width: 600
 
 If you can remove the blade fuses by hand then they are not in completely. Make sure that they are completely seated in the PDP so that they don't pop out during robot operation.
 
 roboRIO swarf
-------------------------------------
+-------------
 
 Swarf is: fine chips or filings of stone, metal, or other material produced by a machining operation. Often modifications must be made to a robot while the control system parts are in place. The circuit board for the roboRIO is conformally coated, but that doesn't absolutely guarantee that metal chips won't short out traces or components inside the case. In this case, you must exercise care in making sure that none of the chips end up in the roboRIO or any of the other components. In particular, the exposed 3 pin headers are a place where chips can enter the case. A quick sweep through each of the four sides with a flashlight is usually sufficient to find the really bad areas of infiltration.
 
-Radio barrel jack
-------------------------------------
+Radio Barrel Jack
+-----------------
 
 Make sure the correct barrel jack is used, not one that is too small and falls out for no reason.  This isn’t common, but ask an FTA and every once in awhile a team will use some random barrel jack that is not sized correctly, and it falls out in a match on first contact.
 
-Ethernet cable
-------------------------------------
+Ethernet Cable
+--------------
 
 If the RIO to radio ethernet cable is missing the clip that locks the connector in, get another cable.  This is a common problem that will happen several times in every competition. Make sure that your cables are secure. The clip often breaks off, especially when pulling it through a tight path, it snags on something then breaks.
 
-Cable slack
-------------------------------------
+Loose Cables
+------------
 
 Cables must be tightened down, particularly the radio power and ethernet cable.  The radio power cables don’t have a lot of friction force and will fall out (even if it is the correct barrel) if the weight of the cable-slack is allowed to swing freely.
 
 Ethernet cable is also pretty heavy, if it’s allowed to swing freely, the plastic clip may not be enough to hold the ethernet pin connectors in circuit.
 
-Reproducing problems in the pit
-------------------------------------
+Reproducing Problems in the Pit
+-------------------------------
 
 Beyond the normal shaking and rattling of all cables while the robot is power and tethered, you might try picking up one side of the robot  off the ground and drop it, and see if you lose connection. The driving on the field, especially when trying to breach defenses will often be very violent. It's better to see it fail in the pit rather than in a critical match.
```

Comment:
```suggestion
Beyond the normal shaking and rattling of all cables whilst the robot is power and tethered, you might try picking up one side of the robot off the ground and drop it, and see if you lose connection. Driving on the field, especially against defenders, will often be very violent. It's better to see it fail in the pit rather than in a critical match.
```